nixos/tests/web-servers/unit-php.nix

```diff
@@ -23,7 +23,10 @@ in {
               "user": "testuser",
               "group": "testgroup",
               "root": "${testdir}/www",
-              "index": "info.php"
+              "index": "info.php",
+              "options": {
+                "file": "${pkgs.unit.usedPhp74}/lib/php.ini"
```

Comment:
You can reference the ini file with the `phpIni` attribute, which should be more future proof.